js/interpreters.js

```diff
@@ -80,8 +80,9 @@ const join = (sources, target, s) => {
     const [aWS, aOrigin, aCorner] = getOriginCornerElement(a);
     const [bWS, bOrigin, bCorner] = getOriginCornerElement(b);
     const [targetWS, targetOrigin, _] = getOriginCornerElement(target);
-    const aDF = aWS.sheet.dataFrame.getDataSubFrame(aOrigin, aCorner);
-    const bDF = bWS.sheet.dataFrame.getDataSubFrame(bOrigin, bCorner);
+    // TODO!
+    const aDF = aWS.sheet.dataStore.getDataSubFrame(aOrigin, aCorner);
```

Comment:
This method is no longer on DataStore. We use plain `getDataArray` with specified coordinate pairs

Reply:
everything related to join has been ignored, it should prob be removed for now